docs/config.rst

```diff
@@ -306,6 +306,12 @@ Complete list of settings that you can put into ``testenv*`` sections:
     such as ``--pre`` (configured as ``pip_pre``) and potentially index-options from the
     deprecated :conf:`indexserver` option.
 
+    .. note::
+
+        You can also provide a bash script to the ``install_command``.
```

Comment:
Let's not single bash out here. Let's say instead arbitrary executable. And that you should ensure the executable can be executed on the OS. For scripts we should encourage to not use scripts, but instead pass the script file to the shell (e.g. bash tox_install.sh is better than tox_install.sh).

Reply:
Good idea! I updated the documentation. I am not sure about `bash tox_install.sh` vs `tox_install.sh` with a proper shebang - I think that is something the users can decide. If there is a very good reason to use the one over the other, you are free to add it :-)